### ISSUE TITLE
Unable to Install `nvidia-dali` in Sessions

### ISSUE THREAD

**Issue by luiscape** [gridai/gridai#99]
## 🐛 Bug

As described in [this issue](https://github.com/NVIDIA/DALI/issues/3218), @danbider is unable to install DALI in a Grid Session. 

### Expected behavior

Able to install `nvidia-dali`.

### Current behavior

Installation fails with the error:

```
WARNING: Ignoring invalid distribution -ygments (/home/jovyan/conda/lib/python3.8/site-packages)

```

### To Reproduce

Steps to reproduce the issue

**Comment by luiscape** [comment]
I managed to install NVIDIA DALI following the tutorial that you mention [here](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/getting_started.html). 

Here are the steps I followed:

1. Start a Session that has a GPU (I started g4dn.xlarge — 1 x T4)
2. Open a terminal and setup conda environment to work with conda init bash — close terminal and open a new one
3. Install dependencies with:

```
pip3 install pip --upgrade && \
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist --upgrade nvidia-dali-cuda110
```

4. Run tutorial

The tutorial worked as expected.
